fix deb description + add Source field

wiiznokes opened the pull request:
I think format will fail because long description is not used anymore, but could be use elsewhere

mdsteele commented:
Do you have a link indicating that long descriptions are no longer used in deb packages?  The multiline portion of `Description` still seems to be listed here: https://www.debian.org/doc/debian-policy/ch-controlfields.html#s-f-description

wiiznokes commented:
> Do you have a link indicating that long descriptions are no longer used in deb packages? The multiline portion of `Description` still seems to be listed here: https://www.debian.org/doc/debian-policy/ch-controlfields.html#s-f-description

Yes your right, mb

wiiznokes commented:
I added long description again, it no longer write `(none)` when empty because it's not necessary.
I also added the command to install the crate when using a custom local version in the read me.

What do you think about testing the `generate_control_file` function? I told myself that it would be more easily testable if it returned a String, which we could compare with a file or a string defined in rust.

Let me know if your interesting by a pr like that
